### DISCUSSION TITLE
Using the Core in android app

### DISCUSSION THREAD

**Discussion by mohsenemx** [v2fly/v2ray-core#2559] [Q&A]
Hi, I'm new to using V2Ray and Xray stuff.
I want to develop my own client for Android.

My question is how to use the core inside an android application?

My preferred language is: Dart+ Flutter

**Comment by the-thirteenth-fox** [comment]
Flutter solution https://github.com/XIIIFOX/flutter_vless - complete free and working nice both iOS and Android
